### ISSUE TITLE
Can't build project due to missing velib library

### ISSUE THREAD

**Issue by mdesjardine** [victronenergy/dbus-adc#2]
This project can't be built because the software/ext/velib library is not in github.

**Comment by jhofstee** [comment]
Indeed, see https://community.victronenergy.com/questions/95566/velib-for-c.html as well. It is not publicly available at the moment.
